test_schemas/en/test_numbers.json

```diff
@@ -291,7 +285,7 @@
                         }
                     ],
                     "id": "test",
-                    "title": ""
+                    "title": "Title"
```

Comment:
Has a card been raised to review the use of group title?

Reply:
I can put one in the Tech Backlog